### DISCUSSION TITLE
拉黑弹幕发送者

### DISCUSSION THREAD

**Discussion by qianbinbin** [the1812/Bilibili-Evolved#4290] [功能建议 / Ideas]
有时看到一些过于逆天的弹幕，点了屏蔽，其实是想拉黑发送者本人，但似乎只屏蔽了弹幕本身

**Comment by the1812** [comment]
建议找别的工具反查弹幕发送者

**Comment by qianbinbin** [comment]
如果一个脚本可以解决问题，又何必安装多个脚本？
https://github.com/qianjiachun/bilibili-danmaku-tracker
